application/service_sync/dos_data.py

```diff
@@ -149,10 +149,17 @@ def save_demographics_into_db(connection: connection, service_id: int, demograph
     """
     if demographics_changes:
         # Update the service demographics
-        logger.info(f"Demographics changes found for service id {service_id}")
-        query = SQL("""UPDATE services SET {} WHERE id = %(SERVICE_ID)s;""").format(
-            SQL(", ".join(f"{key} = '{value}'" for key, value in demographics_changes.items()))
+        logger.info(
+            f"Demographics changes found for service id {service_id}",
+            extra={"demographics_changes": demographics_changes},
         )
+        columns_and_values = [
+            SQL("{} = {}").format(Identifier(key), Literal(value)).as_string(connection)
+            for key, value in demographics_changes.items()
+        ]
+        print(columns_and_values)
```

Comment:
could remove print statement

Reply:
I've removed this print statement used for debugging